### ISSUE TITLE
Git as storage engine

### ISSUE THREAD

**Issue by Kerumen** [lra/mackup#1038]
Is it possible to add a git repository as a storage engine ?

**Comment by kfinlay** [comment]
Follow the instructions for [Configuration > Storage > File System](https://github.com/lra/mackup/blob/master/doc/README.md), then set up git as you would for any project in specified directory.

**Comment by Kerumen** [comment]
I guess you didn't understand my question? Or I wasn't clear? Let me rephrase:
I want to backup my files using `mackup` on a git repository. Not on Dropbox, Google Drive or whatsoever.

**Comment by kfinlay** [comment]
I have it set up the same way. And I do tend to backup less using git than I would using Dropbox for the same reason. But mackup doesn't run as a process, and I doubt many people would want it taking over git. But you could write a shell script to run on a regular basis to commit and push your changes.

**Comment by kennyparsons** [comment]
for anyone here wanting "auto commit" for git/file system engines like I do (sorry, @lra. I do want autocommit and git. No hard feelings though):

set up an alias or function in your shell to run the backup and then commit to your git repo.

**Comment by royteeuwen** [comment]
Anyone made anything automated for this? My company prohibits the usage of personal storage things like DropBox or Google Drive, so I'm stuck with GIT ;)

**Comment by joshmedeski** [comment]
Has anyone tried something like https://github.com/gitwatch/gitwatch? I personally haven't but it could be the solution you're looking for.
